rt/features/logging/src/main/java/org/apache/cxf/ext/logging/event/DefaultLogEventMapper.java

```diff
@@ -46,17 +44,25 @@
 import org.apache.cxf.ws.addressing.ContextUtils;
 
 public class DefaultLogEventMapper {
-    private static final Set<String> BINARY_CONTENT_MEDIA_TYPES;
+    private static final Set<String> DEFAULT_BINARY_CONTENT_MEDIA_TYPES;
     static {
-        BINARY_CONTENT_MEDIA_TYPES = new HashSet<>();
-        BINARY_CONTENT_MEDIA_TYPES.add("application/octet-stream");
-        BINARY_CONTENT_MEDIA_TYPES.add("application/pdf");
-        BINARY_CONTENT_MEDIA_TYPES.add("image/png");
-        BINARY_CONTENT_MEDIA_TYPES.add("image/jpeg");
-        BINARY_CONTENT_MEDIA_TYPES.add("image/gif");
+        DEFAULT_BINARY_CONTENT_MEDIA_TYPES = new HashSet<>();
+        DEFAULT_BINARY_CONTENT_MEDIA_TYPES.add("application/octet-stream");
+        DEFAULT_BINARY_CONTENT_MEDIA_TYPES.add("application/pdf");
+        DEFAULT_BINARY_CONTENT_MEDIA_TYPES.add("image/png");
+        DEFAULT_BINARY_CONTENT_MEDIA_TYPES.add("image/jpeg");
+        DEFAULT_BINARY_CONTENT_MEDIA_TYPES.add("image/gif");
     }
     private static final String MULTIPART_CONTENT_MEDIA_TYPE = "multipart";
 
+    private final Set<String> binaryContentMediaTypes = new HashSet<>(DEFAULT_BINARY_CONTENT_MEDIA_TYPES);
+
+    public void setBinaryContentMediaTypes(String mediaTypes) {
```

Comment:
Maybe it'd be better to call this "addBinaryContentMediaTypes", as it adds to what's already there, as opposed to setting the values.

Reply:
Correct this